### ISSUE TITLE
Keyboard access lesson Color contrast

### ISSUE THREAD

**Issue by smh243** [Section508Coordinators/DHSA-TT-200-A#109]
If you haven't already, please review the [Guidelines for Contributing](contributing.md) in the main folder of this repository before submitting any issues.

## Issue Type (put an X between the brackets to indicate issue type)
- [ ] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [ x] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
[Description of the issue]

## Location
https://training.section508testing.net/mod/book/view.php?id=1229&chapterid=2675
Lesson 1: Keyboard Access
Passing examples Example 1
## Actual Behavior / Observation
The text on the graphic of ANDI under Accessibility Alerts fails color contrast.

**Comment by drewnielson** [comment]
Good thinking in general. However, we would come at this in a couple different ways; the most direct is that we provide an accessible alternative to the image in the form of the text that provides a detailed description of the image just before the image (which you'll cover in Topic 1, which comes at the end of the course).
